week-1/L-functions-nested/exercise2.js

```diff
@@ -1,5 +1,20 @@
+function greeting() {
+  var hello = "Hello";
+  return hello.toUpperCase();
```

Comment:
you do not need to create a var hello make it simple and just return Hello as a text inside quotation

Reply:
okay.